### DISCUSSION TITLE
2022-06-15 Weekly Meeting 60 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#242] [Weekly Meetings]
Weekly Meeting 60 is happening on Wednesday June 15th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 

- Our first blog post is out! https://omigroup.org/1-year-of-omi/

Lean Coffee? Any demos from the community?

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

**Comment by robertlong** [comment]
# Demo Day
- Plan another demo day for mid-July during the week

**Comment by robertlong** [comment]
# Scripting Group
- Demo of Unity -> glTF -> Three.js workflow shared by memelotsqui
- @robertlong did a deep dive on Third Room's multithreading architecture and potential runtime glTF API

**Comment by madjin** [comment]
Interesting video from gltf-extensions group:
- https://www.youtube.com/watch?v=AQUtL_Y3NF8

Media group:
- discussed achievements made since we started
- talked about short / long term strategy
  - short term: optimize what we have, challenge with resources (evo/jin working on it)
  - evo: on doing the narrative work on why those standards matter to cross-industry collaboration we wove this into the MetaTraversal play storyline, and that vid is now up at https://www.youtube.com/watch?v=CYW8Avx48MA
- have divided content creation vs automation process
- now have tweet shift, disseminate from twitter back into community
  - not much content going out atm on that account
    - if people are tweeting, if you don't mind OMI pulling / republishing that'd be good
    - issue is that shift doesn't have filter to sort from OMI related and non-related
    - goes into channel
 #[🔗｜omi-media-links](https://discord.com/channels/770382203782692945/880136862952489000)
 - if anyone wants to participate on a social channel, be an ambassador, chime in the channel

**Comment by robertlong** [comment]
# glTF Extensions Group
- Notes available [here](https://github.com/omigroup/gltf-extensions/discussions/89)

**Comment by madjin** [comment]
- Stardust is doing a bit of a “state of the union” next week via Twitter spaces: https://twitter.com/i/spaces/1PlJQamQqryJE
  - looking for video prototype, need help with motion gfx, video stream where nova has 3D avatar to show up as, figure out right platform to show demo / stream in novas avatar and helper to do a bit of a debut of the look / feel of stardust and what it will look like
- Epic / Unreal do these livestreams that can be a good reference
  - VRChat also, livestream video would make sense there
    - https://www.youtube.com/watch?v=QBQnCxtDaso

**Comment by robertlong** [comment]
# M3 Avatar Interop Group
- Jin shared some updates
- https://github.com/AtlasFoundation/AvatarCreator
- https://github.com/madjin/vrm-samples

**Comment by robertlong** [comment]
# W3C Immersive Web
- We went over the new charter proposal https://w3c.github.io/immersive-web-wg-charter/proposed-immersive-web-wg-charter-202205.html
